Allow for custom PIN length

axemorgan opened the pull request:
- The PIN length can be changed by overriding AppLockActivity.getPinMaxLength() to return the desired PIN size.  The size defaults to 4.
- PinCodeRoundView now adds circle images dynamically based on PIN length, and will shrink their size once they start to overflow the screen width.
- Added PIN length argument to strings which mentioned "4-digit Pincode"

axemorgan commented:
Sorry it took so long.  I made the changes and it's ready to go.

StoyanD commented:
Cool man! I merged and updated the maven repo with build 1.3.1, should be live in a few hours. Also added information in the readme so people know about your feature and added you to the credits

axemorgan commented:
Awesome!  It was my pleasure to contribute back to your project.

On Wed, Jul 22, 2015 at 12:44 PM Stoyan Dimitrov <notifications@github.com>
wrote:

> Cool man! I merged and updated the maven repo with build 1.3.1, should be
> live in a few hours. Also added information in the readme so people know
> about your feature and added you to the credits
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrangeGangsters/LolliPin/pull/43#issuecomment-123786814>
> .
>
